tooling/kb/kb.py

```
@@ -280,6 +280,27 @@ def build(model_name):
 
 
+
+def _require_index():
+    """Clear rebuild hint (not a stack trace) when the local index is missing or schema-stale.
+    kb.db/kb_emb.npy are gitignored/build-on-demand, so a fresh checkout is unbuilt and a local
+    build goes stale as content or this file's schema changes."""
+    hint = ("  run:  cd shipyard && pnpm kb build\n"
+            "  (kb.db/kb_emb.npy are gitignored/build-on-demand; rebuild after a kb.py schema\n"
+            "   change or after ADR/backlog/knowledge content changes.)")
+    if not DB_PATH.exists() or not EMB_PATH.exists():
+        print("  KB index not built yet.\n" + hint); sys.exit(2)
```

Ruff (0.15.20)

[error] Multiple statements on one line (semicolon)

(E702)